src/ddmd/nspace.h

```diff
@@ -1,10 +1,11 @@
-
-// Compiler implementation of the D programming language
-// Copyright: Copyright (c) 2014 by Digital Mars, All Rights Reserved
-// Authors: Walter Bright, http://www.digitalmars.com
-// License: http://boost.org/LICENSE_1_0.txt
-// Source: https://github.com/dlang/dmd/blob/master/src/nspace.h
-
+/**
+ * Compiler implementation of the
+ * $(LINK2 http://www.dlang.org, D programming language).
+ *
+ * Copyright:   Copyright (c) 1999-2017 by Digital Mars, All Rights Reserved
+ * Authors:     $(LINK2 http://www.digitalmars.com, Walter Bright)
+ * License:     $(LINK2 http://www.boost.org/LICENSE_1_0.txt, Boost License 1.0)
+ */
```

Comment:
I know that this is a C header file and not part of our documentation, but there were only two header files using this comment header style, so I thought unifying it makes sense. At some point, we will probably generate these headers anyways...